plugins/workbench-resources/src/components/SavedView.svelte

```diff
@@ -192,7 +193,15 @@
         label: view.string.AddSavedView,
         icon: IconAdd,
         action: async (_, e): Promise<void> => {
-          showPopup(SelectPopup, { value, searchable: true }, eventToHTMLElement(e as MouseEvent), pushMeToFV)
+          showPopup(
+            SelectPopup,
+            { value, searchable: true },
+            getPopupPositionElement(eventToHTMLElement(e as MouseEvent), {
```

Comment:
Alternatively, we could just use `"center"` here to place the component in the center of the viewport (like the + button on Your Projects does), but since the component is a tiny popup here, it looked lonely and weird in the center of the viewport, so I placed it closer to where the click happened

Reply:
ok, let it be on button for now. Thanks.